### PR TITLE
Update test-runtime/project.json for code coverage

### PR DESCRIPTION
**Pull request by iamjasonp**
Our code coverage data has been busted for a while, and it looks like it's due to a tools version update for OpenCover. Update this version so we can get code coverage reports again

### CODE REVIEW

**Comment by iamjasonp** [comment]
@dotnet-bot test code coverage please

**Comment by roncain** [comment]
LGTM, subject to code coverage CI passing.

**Comment by iamjasonp** [comment]
cc: @StephenBonikowsky

**Comment by iamjasonp** [comment]
@dotnet-bot test code coverage please

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -109,7 +109,7 @@ def project = 'dotnet/wcf'
 // **************************
 
 // Define build string
-def codeCoverageBuildString = '''build.cmd /p:ShouldCreatePackage=false /p:ShouldGenerateNuSpec=false /p:OSGroup=Windows_NT /p:Configuration=Windows_NT_Debug /p:Coverage=true /p:WithCategories=\"\\\"InnerLoop;OuterLoop\\\"\"'''
```
Comment (iamjasonp) [inline netci.groovy]: I got a little overzealous in escaping the commands - in Windows it's not necessary to have this escaping, hence this change is needed as well.

**Comment by iamjasonp** [comment]
Successfully tested: http://dotnet-ci.cloudapp.net/job/dotnet_wcf/job/code_coverage_windows_prtest/3/

(this includes the netci.groovy change that was necessary as well)

**Pull request merged**